drivers/SmartThings/sonos/src/api/sonos_connection.lua

```diff
@@ -358,19 +365,28 @@ function SonosConnection.new(driver, device)
         local household_id, current_coordinator =
           self.driver.sonos:get_coordinator_for_device(self.device)
         local _, player_id = self.driver.sonos:get_player_for_device(self.device)
-        self.driver.sonos:update_household_info(header.householdId, body, self.device)
+        self.driver.sonos:update_household_info(header.householdId, body, self.driver)
```

Comment:
this change seems odd, but looking at the function definition change, it looks like the device was just an extra arg before?